tutorial/source/intro_part_i.ipynb

```diff
@@ -110,36 +73,17 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Variable containing:\n",
-      "-0.8290\n",
-      "[torch.FloatTensor of size 1]\n",
-      "\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "x = pyro.sample(\"my_sample\", dist.normal, mu, sigma)\n",
+    "x = pyro.sample(\"my_sample\", dist.Normal(mu, sigma))\n",
     "print(x)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Just like a direct call to `dist.normal`, this returns a sample from the unit normal distribution. The crucial difference is that this sample is _named_. Pyro's backend uses these names to uniquely identify sample statements and _change their behavior at runtime_ depending on how the enclosing stochastic function is being used. As we will see, this is how Pyro can implement the various manipulations that underlie inference algorithms.\n",
-    "\n",
-    "We've seen how `pyro.sample` interacts with a primitive stochastic function like `dist.normal`. Things are no different for an arbitrary stochastic function `fn()`. For example we might have:\n",
-    "\n",
-    "```python\n",
-    "x = pyro.sample(\"my_sample\", fn, arg1, arg2)\n",
-    "```\n",
-    "\n",
-    "This results in a named sample from `fn` with value `x` and arguments `arg1` and `arg2` passed to `fn`."
+    "Just like a direct call to `dist.Normal`, this returns a sample from the unit normal distribution. The crucial difference is that this sample is _named_. Pyro's backend uses these names to uniquely identify sample statements and _change their behavior at runtime_ depending on how the enclosing stochastic function is being used. As we will see, this is how Pyro can implement the various manipulations that underlie inference algorithms."
```

Comment:
nit: I guess this should now read

> Just like a direct call to `dist.Normal(...).sample()`, this returns ...

Reply:
fixed but technically the same since `Normal()()` calls `Normal().sample()`